CONTRIBUTING.md

```diff
@@ -9,12 +9,12 @@ are important to the project's success.
 ## The contribution process at a glance
 
 1. Read the [README.md file](README.md).
-2. Set up your environment to be able to run all tests with `runtests.sh`.  They should pass.
+2. Set up your environment to be able to [run all tests](README.md#running-the-tests).  They should pass.
 3. [Prepare your changes](#preparing-changes):
     * [Contact us](#discussion) before starting significant work.
     * IMPORTANT: For new libraries, [get permission from the library owner first](#adding-a-new-library).
     * Create your stubs [conforming to the coding style](#stub-file-coding-style).
-    * Make sure `runtests.sh` passes cleanly on Mypy, pytype, and flake8.
+    * Make sure your tests pass cleanly on Mypy, pytype, and flake8.
```

Comment:
Should this be across the board for the readme and contributing files? It appears in other places as "Mypy".